SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -37,9 +42,14 @@ void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& Property
 
 	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bUsingQBI))
 	{
-		FMessageDialog::Open(EAppMsgType::Ok,
-			FText::FromString(FString::Printf(TEXT("If you are not using auto-generated launch config, you must make sure to set the value of the \"enable_chunk_interest\" field to \"%s\" in your launch configuration for this to work. (You can check what launch configuration you are using in the SpatialOS GDK for Unreal Editor Settings.)"),
+		const EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo,
+			FText::FromString(FString::Printf(TEXT("If you are not using auto-generated launch config, you must make sure to set the value of the \"enable_chunk_interest\" field to \"%s\" in your launch configuration for this to work. Do you want to configure your launch config settings now?"),
```

Comment:
I'm not sure this makes sense in terms of what a user would need to do in each case. Maybe something along the lines of:
"You must make sure to set the value of the `enable_chunk_interest` Legacy flag to `[true/false]` in your launch configuration for this to work.

If you are using auto-generated launch config, this can be changed from the `SpatialOS GDK for Unreal - Editor Settings` 

Do you want to configure your launch config settings now?"